src/views/Designs.vue

```diff
@@ -81,9 +87,9 @@
                     :size="15"
                   ></v-progress-circular>
                 </td>
-                <td>{{ props.item.design.reaction_knockins.length }}</td>
-                <td>{{ props.item.design.reaction_knockouts.length }}</td>
-                <td>{{ props.item.design.gene_knockouts.length }}</td>
+                <td>{{ props.item.design.reactionKnockins.length }}</td>
+                <td>{{ props.item.design.reactionKnockouts.length }}</td>
```

Comment:
Apart from the `props.item` which is required by the Vuetify template, you are tightly coupling this component with the design implementation (see, for example, [this brief article](https://www.martinfowler.com/ieeeSoftware/coupling.pdf)). Concretely, it'd be better to have a property `numReactionKnockouts` on `design` so that, as an example, it'd be easy to change the actual structure from an array to an object in future without breaking a ton of code.

Reply:
Actually that article is not relevant to this case so much (other than talking about coupling). Maybe these are better resources (although I'm sure you know what I mean):

* https://blogs.msdn.microsoft.com/zainnab/2011/05/25/code-metrics-class-coupling/
* https://en.wikipedia.org/wiki/Coupling_%28computer_programming%29

Reply:
This wasn't introduced in this PR, I think it was in #7, maybe better to move the discussion there.

Reply:
Was meant as a general remark, my bad, probably better to start a discussion issue.